src/screens/main.screen.tsx

```diff
@@ -50,6 +50,24 @@ const MainScreen = ({ navigation }: ScreenProps) => {
     }
   }, [locationError]);
 
+  // empty page as default content
+  let mainContent: React.JSX.Element = (
```

Comment:
@havardf perhaps using useState for this?

Reply:
Its not really state, as I dont need that variable between rendering. This just an attempt to avoid having to duplicate all the "outer" JSX components that the page needs, no matter if it shows an error, spinner or an actual forecast.

Reply:
Alright. It looks good to me then.